sway/config/output.c

```diff
@@ -513,125 +504,43 @@ static void default_output_config(struct output_config *oc,
 	oc->max_render_time = 0;
 }
 
-static struct output_config *get_output_config(char *identifier,
-		struct sway_output *sway_output) {
-	const char *name = sway_output->wlr_output->name;
-
-	struct output_config *oc_id_on_name = NULL;
-	struct output_config *oc_name = NULL;
-	struct output_config *oc_id = NULL;
+struct output_config *find_output_config(struct sway_output *sway_output) {
+	// Start with a default config for this output
+	struct output_config *result = new_output_config("merge");
+	default_output_config(result, sway_output->wlr_output);
```

Comment:
Just as a note, this is currently only used on reload. I think it is fine for this to be applied unconditionally, but there may be an unintentional side effect that I'm overlooking.

Reply:
`default_output_config()` itself doesn't have any side effects, it just initializes the struct. Having the same exact code path for a config change and for reload seems like an advantage. Bug #5632 is the kind of thing that can happen when there are two code paths.